test/applocale/ruby_test.rb

```diff
@@ -1,11 +1,7 @@
 require "test_helper"
 
-class Applocale::RubyTest < Minitest::Test
```

Comment:
file is named ruby_test, but we've updated the class name to be VersionTest. Mind renaming the file to version_test.rb?

Reply:
good catch! Fixed.